SparkPost/2web2ui
# FE-1124: Enable Github Code Scanning on 2web2ui

### What Changed
 - Setup a Github action to run CodeQL on master and all pull requests

Helpful Links
 - https://docs.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/automatically-scanning-your-code-for-vulnerabilities-and-errors
 - https://github.com/SparkPost/2web2ui/pull/1726/checks?check_run_id=883490145
 - https://www.helpnetsecurity.com/2020/05/08/github-code-scanning/

### How To Test
_Nothing to see since "Missing analysis for base commit", [see "Code scanning" check](https://github.com/SparkPost/2web2ui/pull/1726/checks?check_run_id=883505888)_
 - Navigate to the "Checks" tab and confirm the action is running
